contrib/win32/openssh/FixUserFilePermissions.ps1

```diff
@@ -14,5 +14,19 @@ Get-ChildItem ~\.ssh\* -Include "id_rsa","id_dsa" -ErrorAction SilentlyContinue
     Repair-UserKeyPermission -FilePath $_.FullName @psBoundParameters
 }
 
+
+if (([bool]([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")))
```

Comment:
Rearrange the if the conditions so users get notified they have to run in elevated mode.


$sshdAdministratorsAuthorizedKeysPath = join-path $env:ProgramData\ssh "administrators_authorized_keys"
if (Test-Path $sshdAdministratorsAuthorizedKeysPath -PathType Leaf)
{
    if (([bool]([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")))
    {
        Repair-AdministratorsAuthorizedKeysPermission -FilePath $sshdAdministratorsAuthorizedKeysPath @psBoundParameters
    }
    else 
    {
        Write-host "To fix file permissions for $sshdAdministratorsAuthorizedKeysPath, run this script in elevated mode"  -ForegroundColor Yellow
    }
}